cu-calendar.sty

```diff
@@ -112,13 +112,13 @@
 }%
 %
 \def\cuDayName#1{%
-  \ifcase#1 воскресение\or%
-    понедельник\or
-    вторник\or
-    среда\or
-    четверг\or
-    пятница\or
-    суббота\fi
+  \ifcase#1 недѣ́лѧ\or%
+    понедѣ́льникъ\or
+    вто́рникъ\or
+    среда̀\or
+    четверто́къ\or
+    пѧто́къ\or
+    сꙋббѡ́та\fi
 }%
```

Comment:
I think also we need only one case. The only question is which case? Will this macro be called in the context of e.g., "Date: \today" (nominative case) or "this book was printed on \today" (accusative case)?

I actually don't think we need the capitalized forms, since capital letters in Church Slavonic are only used at the beginning of a sentence.